.eslintrc.cjs

```diff
@@ -0,0 +1,114 @@
+const nodeFiles = [
+  '**/build.dflt.{js,mjs,cjs}',
+  '**/build.{js,mjs,cjs}',
+  '**/config/**',
+  '**/prebuild.{js,mjs,cjs}',
+  '**/prebuild/**',
+  '**/scripts/**',
+  'packages/new-design/lib/**',
+  'sites/backend/**',
+  'sites/*/mdx/**',
+  'sites/*/themes/**',
+]
+const frontendFiles = [
+  '**/components/**',
+  '**/hooks/**',
+  '**/pages/**',
+  '**/page-templates/**',
+  'packages/i18n/**/*.md/*.js',
+]
+const isomorphicFiles = ['packages/core/**', 'plugins/**', 'tests/designs/**', 'tests/plugins/**']
+
+module.exports = {
+  extends: 'eslint:recommended',
+  env: {
+    es2021: true,
+  },
+  // Required when using experimental EcmaScript features
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    // Options specific to the Babel parser
+    requireConfigFile: false,
+    babelOptions: {
+      plugins: ['@babel/plugin-syntax-import-assertions'],
+    },
+  },
+  rules: {},
+  overrides: [
+    // Partitioned JavaScript files
+    {
+      files: nodeFiles,
+      env: {
+        node: true,
+      },
+      rules: {
+        'no-console': 'off',
+      },
+    },
+    {
+      files: frontendFiles,
+      excludedFiles: nodeFiles,
+      extends: ['next/core-web-vitals'],
```

Comment:
This adds support for JSX syntax and also adds a bunch of React- and Next-specific rules